whitepapers/0004_message_publishing.md

```diff
@@ -102,6 +102,8 @@ Fee uint256
 
 TransferFees:
 
+> **NOTE:** The Wormhole Core Contract does not conform to this spec. The `Amount` and `To` fields are in reverse order. (Ref: [Core Contract](https://github.com/wormhole-foundation/wormhole/blob/main/cosmwasm/contracts/wormhole/src/state.rs#L386-L397))
```

Comment:
Could you narrow this to "CosmWasm Core Contract" please? Other core contract implementations like Solana conform to the spec

Reply:
Sure thing!